pages/card.html

```diff
@@ -0,0 +1,309 @@
+                sandbox="allow-scripts allow-same-origin">
+            </iframe>
+
+
+            <h3 class="heading-small">Card with Text Overlay</h3>
+            <p class="description">Cards With Badges are similar to basic cards, but with badges added to them. You can
```

Comment:
I think here you're supposed to give description for card with text overlay.

Reply:
Yeah. Will modify it. Thanks for pointing it out Parvez.